### ISSUE TITLE
monitors geting frozen when loged out near

### ISSUE THREAD

**Issue by badgeminer** [SquidDev-CC/CC-Tweaked#1428]
### Minecraft Version

1.19.2

### Version

1.101.2

### Details

logs: 
- [https://gist.githubusercontent.com/badgeminer/7a244b8036df1b5ffd15b843b8fe8d7b/raw/36d7fd1a36dce51f6595274b854834c56bca5e3f/latest.log](https://gist.githubusercontent.com/badgeminer/7a244b8036df1b5ffd15b843b8fe8d7b/raw/36d7fd1a36dce51f6595274b854834c56bca5e3f/latest.log)

what i did
- connected 2 advanced monitors to an advanced computer
- ran `set avi.cbn.l monitor_7` and `set avi.cbn.r monitor_8`
- put this code [https://gist.githubusercontent.com/badgeminer/7a244b8036df1b5ffd15b843b8fe8d7b/raw/36d7fd1a36dce51f6595274b854834c56bca5e3f/cbn.lua](https://gist.githubusercontent.com/badgeminer/7a244b8036df1b5ffd15b843b8fe8d7b/raw/36d7fd1a36dce51f6595274b854834c56bca5e3f/cbn.lua) in the startup/ folder on the advanced computer
- relogged

**Comment by badgeminer** [comment]
found on 1.19.2

**Comment by SquidDev** [comment]
I'm afraid I've been unable to reproduce this, either when the computer is in spawn chunks (and so remains on during relogging) or not (and so is restarted due to chunks loading/unloading). Does this happen reliably for you?

**Comment by SquidDev** [comment]
I'm going to close this as I've been unable to reproduce this. If folks have more information, happy to reopen this again.

**Comment by garrowson** [comment]
MC 1.19.2 / Forge 43.2.14 / CC 1.101.3

Happening to me to. Relogging or leaving the area will freeze the monitor.

It starts working again if you replace the monitor or place a second monitor (resizing it) next to it.

The test-program I used (startup.lua)
```
m=peripheral.find("monitor")
m.clear()
while true do
  m.setCursorPos(1,1)
  m.write("I")
  sleep(.5)
  m.setCursorPos(1,1)
  m.write("-")
  sleep(.5)
end
```

**Comment by garrowson** [comment]
Was literally able to reproduce it on a fresh install. Using versions and code from my previous post. 
![image](https://github.com/cc-tweaked/CC-Tweaked/assets/49206998/83157291-e34e-40bc-9a1f-1baca72396c4)

Didn't work within the spawn chunks, But outside I just had to teleport 1000 away and right back: Monitor stuck

**Comment by garrowson** [comment]
@SquidDev hope I'm not annoying. But I don't know if closed issues are monitored.

**Comment by bastiencantet** [comment]
I have the same problem

**Comment by bastiencantet** [comment]
when teleporting / logging out or leaving the area fast will freeze the monitor

**Comment by bastiencantet** [comment]
on server

**Comment by Kan18** [comment]
I am consistently able to reproduce this bug on the SwitchCraft 3 server (CC:T 1.108.3 Fabric 1.20.1), but it seems to happen only in certain areas (maybe something to do with too much chunk data?)
1. Place a computer at or near -919, 69, -122 and place a 4x4 monitor to the right of the computer
2. On the computer, run this program:
```lua
local monitor = peripheral.find("monitor")
local x, y = monitor.getSize()
while true do
    monitor.scroll(1)
    monitor.setCursorPos(1, y)
    monitor.write(os.epoch("utc"))
    sleep(0.85)
end
```
3. Log out and rejoin
4. The monitor becomes frozen

https://github.com/cc-tweaked/CC-Tweaked/assets/24967425/d46de80f-0125-4225-82d7-028602d2d87e

Also, if the monitor is in this state, it can be "warped" by breaking all of its blocks except the bottom left 
<img width="1280" alt="image" src="https://github.com/cc-tweaked/CC-Tweaked/assets/24967425/775408a8-2625-46f3-8d01-e009f8919a89">
(sorry; I intended to put this reply in #1560 but I had the wrong issue open)

**Comment by boddahburnout** [comment]
I am also currently having this issue, I didn't have it before but I'm built closer to spawn chunks this time maybe that's it? I'm using artist with an external display and if anyone tp's away or if everyone logs off the monitors are frozen until they are broke and replaced, terribly annoying. If you want a copy of the world or anymore info lmk because I really want to see this bug squashed
1.19.2

**Comment by Wojbie** [comment]
> I really want to see this bug squashed
1.19.2

This bug was fixed in https://github.com/cc-tweaked/CC-Tweaked/issues/1560 using commit  https://github.com/cc-tweaked/CC-Tweaked/commit/4daa2a2b6a839769c8be31425780e6c75c1bd752 but at that time 1.19.2 was no longer in development.

**Comment by boddahburnout** [comment]
So then the fix is included in 1.2x and up?

**Comment by boddahburnout** [comment]
Ah, yes 1.20.X that's incredibly unfortunate I won't be able to update my server for a good amount of time. Thanks for the reply
